### DISCUSSION TITLE
BeamNG Linux

### DISCUSSION THREAD

**Discussion by letsruletheworld** [Ultrawipf/OpenFFBoard#131] [General]
hi - awesome project. 

got it running and had some initial trouble getting FFB on BeamNG in Linux(steam).

the trick is:

`Once you added your wheel, you will need to enable FFB as it will be disabled by default. If you just leave it enabled without changing anything else, it will not work. You will need to set ''Update Type'' to ''Full (Slow Fallback)''`

source:
[](https://www.beamng.com/threads/beamng-on-archlinux-guide-with-logitech-g-wheels-force-feedback.94833/)

maybe you could add this to nodes on the working games list.

**Comment by Ultrawipf** [comment]
Interesting. I will try to reproduce it if thats the case. 
On windows i can confirm that beamng works with all ffb modes and i would recommend the fast update mode there. 

Possibly because of the different internal ffb effect handling on Linux updates are not forwarded or enabled otherwise. Could be interesting to investigate what the reason is.
